manager/command.py

```diff
@@ -0,0 +1,44 @@
+from os import system
+
+
+class AbstractCommand:
```

Comment:
I can only specify this one as abstract. I have to specify common type at least for 2 objects.